client/packages/views/packages-list.js

```diff
@@ -16,18 +18,28 @@ const noPackages = () => {
 	);
 };
 
-const PackagesList = ( { form, removePackage, editPackage } ) => {
-	const {
-		packages,
-		dimensionUnit,
-	} = form;
+const PackagesList = ( { packages, dimensionUnit, editable, selected, serviceId, groupId, removePackage, editPackage, toggleAll, togglePackage } ) => {
+	const renderSelectAll = () => {
+		if ( ! selected ) {
+			return null;
+		}
+
+		const allPackageIds = packages.map( ( def ) => def.id );
+		const selectedAll = 0 === _.difference( allPackageIds, selected ).length;
+		return <FormLegend className="package-actions"><FormCheckbox checked={ selectedAll } onChange={ () => toggleAll( serviceId, groupId ) }/></FormLegend>;
+	};
 
 	const renderPackageListItem = ( pckg, idx ) => {
+		const isSelected = selected && -1 !== selected.indexOf( pckg.id );
```

Comment:
`Array.prorotype.includes()` is much cleaner than the `-1` thing. Let's make good use of babel :) [Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)